### PR TITLE
[FIX]: fixed links for documentation in README

### PR DESCRIPTION
**Pull request by Pragya2056**
[ISSUE](https://github.com/erxes/erxes/issues/3614)

### Context

Deleted the links which led to 404 errors

// Delete the below section once completed
### PR Checklist
- [x] Description is clearly stated under Context section
- [ ] Screenshots and the additional verifications are attached

### CODE REVIEW

**Comment by Pragya2056** [comment]
@naurenbatjargal 
Please review this.

**Comment by Pragya2056** [comment]
Someone please review this.

File: README.md [hunk 1]
```diff
@@ -50,7 +50,7 @@
 </p>
 
 <p align="center">
- <a href="https://www.erxes.io/![Cover](https://erxes-docs.s3.us-west-2.amazonaws.com/xos.jpeg)"><img src="https://erxes-docs.s3.us-west-2.amazonaws.com/xos.jpeg" width="100%" alt="erxes: Free and open fair-code licensed experience operating system (XOS)">
+ <a href="https://erxes.io/static/images/home/home1.png"><img src="https://erxes-docs.s3.us-west-2.amazonaws.com/xos.jpeg" width="100%" alt="erxes: Free and open fair-code licensed experience operating system (XOS)">
```
Comment (Anu-Ujin) [inline README.md:53]: @Pragya2056  please change this link to https://erxes-docs.s3.us-west-2.amazonaws.com/xos.jpeg

**Comment by Pragya2056** [comment]
@Anu-Ujin Please review the PR

**Comment by Anu-Ujin** [comment]
> @Anu-Ujin Please review the PR

Can you change the link please. After that i can merge your pr and close the issue

File: README.md (hunk 1 above)
Comment (Pragya2056) [reply README.md]: I made the change

**Comment by Pragya2056** [comment]
> > @Anu-Ujin Please review the PR
> 
> Can you change the link please. After that i can merge your pr and close the issue

It was done.

**Review by Anu-Ujin — APPROVED** [review]
close #3614

**Comment by Anu-Ujin** [comment]
Merged! Thank you for your participation @Pragya2056 :)

**Pull request merged**